test/test_Quaternion.jl

```diff
@@ -1,5 +1,6 @@
 
 using Quaternions: argq
+import DualNumbers
```

Comment:
`import` should be avoided unless overloading

```suggestion
using DualNumbers
```